css/src/tokens/table.scss

```diff
@@ -0,0 +1,7 @@
+$table-border-width: 1px;
```

Comment:
I think these are not really proper tokens, they're variables that should live closer to the table class itself.

Reply:
Moved to the top of the `components/table.scss`